### PR TITLE
python310Packages.testcontainers: init at 3.5.0

### PR DESCRIPTION
**Pull request by onny**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/testcontainers/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  pythonImportsCheck = [
+    "testcontainers"
+  ];
+
+  meta = with lib; {
+    description = ''
+      Lightweight, throwaway instances of common databases, Selenium web
+      browsers, or anything else that can run in a Docker container
+    '';
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/testcontainers/default.nix:48]:
```suggestion
    longDescription = ''
      Lightweight, throwaway instances of common databases, Selenium web
      browsers, or anything else that can run in a Docker container
    '';
```

File: pkgs/development/python-modules/testcontainers/default.nix [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+  # Tests require various container and database services running
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "testcontainers"
+  ];
+
+  meta = with lib; {
+    longDescription = ''
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/testcontainers/default.nix:41]: longDescription should always be accompanied by description.

Comment (SuperSandro2000) [inline pkgs/development/python-modules/testcontainers/default.nix:41]:
```suggestion
    description = "Allows using docker containers for functional and integration testing"
    longDescription = ''
```

**Comment by smancill** [comment]
Result of `nixpkgs-review pr 181584` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python310Packages.testcontainers</li>
    <li>python39Packages.testcontainers</li>
  </ul>
</details>

**Comment by onny** [comment]
Thank you @SuperSandro2000 for reviewing, fixed the missing issues

File: pkgs/development/python-modules/testcontainers/default.nix [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, deprecation
+, docker
+, wrapt }:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/testcontainers/default.nix:6]:
```suggestion
, wrapt
}:
```

File: pkgs/development/python-modules/testcontainers/default.nix [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "testcontainers"
+  ];
+
+  meta = with lib; {
+    description = ''
+      Allows using docker containers for functional and integration testing
+    '';
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/testcontainers/default.nix:35]:
```suggestion
    description = "Allows using docker containers for functional and integration testing";
```

**Pull request merged**